networkit/_NetworKit.pyx

```diff
@@ -4,7 +4,7 @@
 #includes
 # needed for collections.Iterable
 from networkit.exceptions import ReducedFunctionalityWarning
-import collections
+import collections.Iterable
```

Comment:
Shouldn't it be `from collections.abc import Iterable` (or similar)?
 https://docs.python.org/3/library/collections.abc.html